packages/domains-table/src/domains-table/domains-table-status-cell.tsx

```diff
@@ -47,8 +50,9 @@ export const DomainsTableStatusCell = ( {
 	};
 
 	const getTime = ( unixTimestamp: number ) => {
-		const date = new Date( unixTimestamp * 1000 );
-		return `${ date.toLocaleDateString() } ${ date.toLocaleTimeString() }`;
+		const date = formatDate( locale, unixTimestamp * 1000 );
+		const time = new Date( unixTimestamp * 1000 ).toLocaleTimeString();
+		return `${ date } ${ time }`;
```

Comment:
_Technically_ concatenating dates and time strings together with a space in-between is a locale-specific setting too right?

Why not use `Intl.DateTimeFormat( localeSlug, { dateStyle: 'medium', timeStyle: 'medium' } ).format()` so that it does the fully localised date-time string?

Or ignore me if I'm overthinking it.

Reply:
Right. My final solution feels a bit clumsy. I figured putting everything in one function was good, but we don't need the time in one place. Maybe better to remove `dates.ts` and just use `Intl.DateTimeFormat` separately. I will do that.